### ISSUE TITLE
"__dockview_internal_drag_event__" is pasted if drop on an editable widget

### ISSUE THREAD

**Issue by hackwaly** [mathuo/dockview#584]
Drag a tab to chrome's location bar, it will replace the location to "__dockview_internal_drag_event__"

**Comment by techbech** [comment]
I wanted to make it possible to drag a tab onto another application, so I requested the possibility to access the data transfer object

https://github.com/mathuo/dockview/issues/299

It was implemented here

https://github.com/mathuo/dockview/pull/322/files#diff-a22c4f0fca5755287b9458a40efb6520028ddad7a4224ab7f83ae524900551afR86-R114

This way, you can make the value, when dropped in a text field, whatever you like.

**Comment by mcapodici** [comment]
Thanks @techbech, based on your sample, sticking this in `onReady` seems to work around the issue. Nice if this were a configuration option to begin with though.

```typescript
  api.onWillDragPanel((event) => {
    const dataTransfer = event.nativeEvent.dataTransfer;

    if (dataTransfer) {
      dataTransfer.setData("text/plain", "");
    }
  });
```

For some reason `dataTransfer.clear()` didn't work.

**Comment by mathuo** [comment]
Released in `2.0.0`. See https://github.com/mathuo/dockview/releases/tag/v2.0.0
